sphinxcontrib/test_reports/directives/test_case.py

```diff
@@ -148,6 +149,18 @@ def run(self, nested=False, suite_count=-1, case_count=-1):
         if case_parameter is None:
             case_parameter = ""
 
+        # Set extra data, which is not part of the Sphinx-Test-Reports default options
+        for key, value in case.items():
```

Comment:
does any code part do input validation of the input json, e.g. is there a schema that checks the value is actually a string?